src/sugar/autofill.ts

```diff
@@ -10,8 +10,7 @@ import { xrpToDrops } from '../utils'
 
 // 20 drops
```

Comment:
This comment is incorrect. The LEDGER_OFFSET is actually used to set the default for `LastLedgerSequence`, basically, how many ledgers can be validated before a transaction expires. The default of 20 ledger versions is ~1 minute given typical ledger close times of ~3 seconds.

```suggestion
// Expire unconfirmed transactions after 20 ledger versions, approximately 1 minute, by default
```